anaconda-pkg-build/linux/entrypoint.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -o errtrace -o nounset -o pipefail -o errexit -x
+
+env | sort
+
+DEFAULT_BUILD_ARGS=(--suppress-variables --error-overlinking --error-overdepending)
+declare -a BUILD_ARGS=()
+for arg do
+    shift
+    if [[ "${arg}" == -build-args=* ]]; then
+        IFS=',' read -r -a BUILD_ARGS <<< "${arg//-build-args=}"
+        continue
+    fi
+    set -- "$@" "${arg}"
+done
+
+if [ "${#BUILD_ARGS[@]}" -eq 0 ]; then
+    BUILD_ARGS="${DEFAULT_BUILD_ARGS[@]}"
```

Comment:
shellcheck does not like that line... requires to be changed to the correct code.